server/service/src/preference/query_preference.rs

```diff
@@ -0,0 +1,96 @@
+use repository::{
+    EqualFilter, PreferenceFilter, PreferenceRepository, PreferenceRow, RepositoryError,
+    StorageConnection,
+};
+
+pub fn query_global(
```

Comment:
separate helpers for handling global prefs, store prefs (would extend when adding other pref types)

These are used as defaults (see the types.rs) but can also implement your own if the pref query was custom for whatever reason